### PR TITLE
Implement pip support

### LINKED ISSUE #26: Support pip-based installations

### PR DESCRIPTION
**Pull request by ericof**
Closes #26

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Changelog
+
```
Comment (davisagli) [inline CHANGELOG.md:2]: @ericof It would probably be best to add a note here to remind people to use towncrier

File: setup.cfg [hunk 2]
```diff
@@ -5,12 +5,3 @@ ignore =
     .editorconfig
     .gitattributes
     bootstrap-buildout.py
```
Comment (davisagli) [inline setup.cfg]: @ericof the check-manifest config can also go in pyproject.toml: https://github.com/mgedmin/check-manifest#configuration

File: src/kitconcept/dsgvo/configure.zcml [hunk 3]
```diff
@@ -47,72 +50,74 @@
...
     <adapter
+        factory=".userdataschema.DsgvoUserDataSchemaAdapter"
         provides=".userdataschema.IDsgvoUserDataSchema"
         for="plone.app.layout.navigation.interfaces.INavigationRoot"
-        factory=".userdataschema.DsgvoUserDataSchemaAdapter"
         />
 
   </configure>
 
   <!-- Plone 4 -->
```
Comment (davisagli) [inline src/kitconcept/dsgvo/configure.zcml]: The zcml for Plone 4 could be removed, right?

File: src/kitconcept/dsgvo/profiles/common/memberdata_properties.xml [hunk 4]
```diff
@@ -1,6 +1,14 @@
...
+>
+  <property name="dsgvo_accept"
+            type="boolean"
+  >False</property>
+  <property name="dsgvo_registration_date"
+            type="date"
+  >2012/12/01</property>
+  <property name="dsgvo_registration_ip"
+            type="string"
+  />
```
Comment (davisagli) [inline src/kitconcept/dsgvo/profiles/common/memberdata_properties.xml:13]: zpretty's style really makes things harder to read and harder to write when the lines are this short

**Review by davisagli — APPROVED** [review]
I made a few suggestions for further cleanup, but LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (ericof) [reply CHANGELOG.md:2]: @davisagli Done

**Review by davisagli — APPROVED** [review]

**Pull request merged**